elastic/kibana
# [Synthetics Integration] add enabled toggle

## Summary

Relates to https://github.com/elastic/kibana/issues/119519

Adds enabled Toggle to the Synthetifcs Integration UI.

<img width="718" alt="Screen Shot 2021-11-30 at 10 32 19 AM" src="https://user-images.githubusercontent.com/11356435/144094845-3c41e329-ac12-440c-84f2-149a0724e089.png">

> ⚠️ This PR depends on the integration in elastic/integrations#2290

### Testing
**⚠️ NOTE: You'll need to install the `synthetics` integration version in elastic/integrations#2290 to be able to test this. To do so, you can use `elastic-package` to `build` it, as per [the instructions on their repo](https://github.com/elastic/elastic-package).**
1. Checkout the above integrations PR. Build the synthetics package by navigating to the `packages/synthetics` directory and running `elastic-package build`
2. Ensure that you do not have an existing instance of ES up and running. Run `env ELASTICSEARCH_IMAGE_REF=8.1.0-SNAPSHOT ELASTIC_AGENT_IMAGE_REF=8.1.0-SNAPSHOT KIBANA_IMAGE_REF=8.1.0-SNAPSHOT elastic-package stack up -d -v --version 8.1.0-SNAPSHOT`
3. Run `curl -u elastic:changeme -X POST "http://localhost:9200/_security/user/kibana_system/_password?pretty" -H 'Content-Type: application/json' -d' { "password" : "changeme" } '`
4. Checkout this local branch of Kibana. Add the following to `kibana.dev.yml`
```
kibana.port: 5602
xpack.fleet.registryUrl: "http://localhost:8080"
```
5. Start kibana
5. Log into `localhost:5602`
6. Navigate to the Elastic Synthetics package and add an integration. Observe the enabled toggled is checked.
7. Create a monitor with the enabled toggled check. Navigate to Uptime to view the monitor results.
8. Return to that monitor and uncheck the toggled switch. Navigate back to Uptime to view that heartbeat has stopped indexing documents. 

### Checklist

Delete any items that are not applicable to this PR.
